types/jquery/misc.d.ts

```diff
@@ -1205,7 +1182,7 @@ callbacks.fire( "world" );
     interface PromiseBase<TR, TJ, TN,
         UR, UJ, UN,
         VR, VJ, VN,
-        SR, SJ, SN> extends _Promise<TR>, PromiseLike<TR> {
+        SR, SJ, SN> {
```

Comment:
Arguably, we might want to keep `extends PromiseLike`:

```suggestion
        SR, SJ, SN> extends PromiseLike<TR> {
```